src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -388,7 +388,7 @@ ServerHandlerImp::processSession(
     auto is = std::static_pointer_cast<WSInfoSub> (session->appDefined);
     if (is->getConsumer().disconnect())
     {
-        session->close();
+        session->close(1008, "threshold exceeded");
```

Comment:
Instead of the magic number `1008` here, could we use `boost::beast::websocket::policy_error`?